chapters/optimization/index.md

```diff
@@ -0,0 +1,155 @@
+## Flow Optmization
```

Comment:
I think in the new template we need a summary of the topics that will be covered in the doc.

You can add it just like this in rst, not sure how this will be with markdown

.. contents:: Table of Contents
    :depth: 2